java/AxivityReader.java

```diff
@@ -18,19 +20,25 @@
  */
 public class AxivityReader extends DeviceReader {
 
-    // public static ZonedDateTime SESSION_START = null;
-    // public static ZoneId zoneId = ZoneId.of("Europe/London");
-    public static LocalDateTime SESSION_START = null;
```

Comment:
Now `SESSION_START` is a zoned time and its DST status `sessionStartDST` will be registered. `sessionStartDST` is important as all subsequent timestamps' DST status compared against to see if a crossover has occurred.